src/vs/workbench/services/workspaces/common/workspaceTrust.ts

```diff
@@ -127,7 +121,7 @@ export class WorkspaceTrustManagementService extends Disposable implements IWork
 		await this.updateWorkspaceTrust();
 	}
 
-	private calculateWorkspaceTrust(): boolean {
+	protected calculateWorkspaceTrustDefault(): boolean | undefined {
```

Comment:
I don't think default is the right word to use here, default implies to me something like a fallback, but this is the opposite of that. Since this isn't a calculation against the workspace URIs, maybe `getForcedWorkspaceTrust(): boolean | undefined`

Reply:
This has been refactored.